ssr/render.js

```diff
@@ -232,7 +232,6 @@ export default function render(
     }
   }
 
-  $("title").text(pageTitle);
```

Comment:
This change is unrelated. 

When I was testing out this PR, I wanted to make some/any edit to this `.js` file, just to see that it's reflected once I refresh the browser. So I messed with the title and I noticed that we're setting the `$('title').text` twice.